dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -152,66 +151,6 @@ export default defineComponent({
         this.listOfDataSetReportingPeriods as ReportingPeriodOfDataSetWithId[]
       );
     },
-
-    /**
-     * Converts a number to millions with max two decimal places and adds "MM" at the end of the number.
-     * @param inputNumber The number to convert
-     * @returns a string with the converted number and "MM" at the end
-     */
-    convertToMillions(inputNumber: number): string {
-      return `${(inputNumber / 1000000).toLocaleString("en-GB", { maximumFractionDigits: 2 })} MM`;
-    },
-
-    /**
-     * Converts a nace code to a human readable value
-     * @param kpiValue the value that should be reformated corresponding to its field
-     * @returns the reformatted Country value ready for display
-     */
-    reformatIndustriesValue(kpiValue: KpiValue) {
-      return Array.isArray(kpiValue)
-        ? kpiValue.map((naceCodeShort: string) => naceCodeMap.get(naceCodeShort)?.label ?? naceCodeShort)
-        : naceCodeMap.get(kpiValue as string)?.label ?? kpiValue;
-    },
-
-    /**
-     * Converts a country code to a human readable value
-     * @param kpiValue the value that should be reformated corresponding to its field
-     * @returns the reformatted Country value ready for display
-     */
-    reformatCountriesValue(kpiValue: KpiValue) {
-      return Array.isArray(kpiValue)
-        ? kpiValue.map(
-            (countryCodeShort: string) => getCountryNameFromCountryCode(countryCodeShort) ?? countryCodeShort
-          )
-        : getCountryNameFromCountryCode(kpiValue as string) ?? kpiValue;
-    },
-
-    /**
```

Comment:
This method should remain framework specific such that there is no reformatting done for one framework which is not desired in a different one.

Reply:
Also maybe add a description for this method

Reply:
Which method exactly should not be extracted?

Reply:
`reformatValueForDisplay`. We should not use the same version of this method for each framework.

Reply:
Lets discuss it in a call